src/routes/safe/components/Layout/Tabs/index.jsx

```diff
@@ -100,7 +100,7 @@ const TabsComponent = (props: Props) => {
         label={labelTransactions}
         value={`${match.url}/transactions`}
       />
-      {!process.env.REACT_APP_APPS_DISABLED && (
+      {process.env.REACT_APP_APPS_DISABLED !== 'true' && (
```

Comment:
I would change it by `!!process.env.REACT_APP_APPS_DISABLED`

Reply:
why is that? using `!!` will always evaluate to true:
```
!!'true'
> true
!!'false'
> true
```

Reply:
I think it is not set if you don't want it

Reply:
it's set in all the environments with 'false' value